### PR TITLE
Expose Path, Neighbors, Packages and Sources query

### PR DESCRIPTION
**Pull request by pxp928**
For `Path` and `Neighbors` queries `fragment allNodes` do not work as it causes an error in the generated code.

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

**Review by lumjjb — APPROVED** [review]
lgtm

File: pkg/assembler/clients/operations/path.graphql [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    ... on Builder {
+      ...allBuilderTree
+    }
+  }
+}
+
+query Neighbors($node: ID!) {
+  neighbors(node: $node) {
+    __typename
```
Comment (lumjjb) [inline pkg/assembler/clients/operations/path.graphql:88]: qq: what is the typename here again?

**Review by lumjjb — APPROVED** [review]

File: pkg/assembler/clients/operations/path.graphql (hunk 1 above)
Comment (pxp928) [reply pkg/assembler/clients/operations/path.graphql:88]: that's how graphQL knows how to parse: 	Typename *string `json:"__typename"`

Comment (mihaimaruseac) [reply pkg/assembler/clients/operations/path.graphql:88]: It's the node name, the name from GraphQL (so, Package, Artifact, etc.) This is how you know what type was actually used in the union

**Pull request merged**